invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentItems/DocumentItems.js

```diff
@@ -21,12 +22,11 @@ export default class DocumentItems extends Component {
     this.fetchDocumentItems(this.props.document.document_pid);
   }
 
-  showDetailsHandler = itemPid =>
-    this.props.history.push(this.showDetailsUrl(itemPid));
+  showDetailsHandler = itemPid => goTo(this.showDetailsUrl(itemPid));
```

Comment:
We can skip that declaration, and use directly `goToHandler` further down where it is used.